includes/Core/Assets/Assets.php

```diff
@@ -404,6 +404,16 @@ private function get_assets() {
 					'dependencies' => $dependencies,
 				)
 			),
+			new Script(
+				'googlesitekit-site',
+				array(
+					'src'          => $base_url . 'js/googlesitekit-site.js',
```

Comment:
That file seems to not actually be there. See above, it should just register the store.

Reply:
Yup, I messed up the webpack commit locally and didn't push it. I'll get that fixed 👍